AssetRipperGUI/MainWindow.ViewModel.cs

```diff
@@ -266,5 +282,63 @@ public void OnAssetSelected(NewUiFileListItem selectedItem, Object selectedAsset
 			SelectedAsset?.Dispose();
 			SelectedAsset = new(selectedAsset, _assetContainer);
 		}
+		
+		// Called from UI
+		private async void CheckforUpdates()
+		{
+			const string url = "https://api.github.com/repos/ds5678/AssetRipper/releases";
+			string json = await client.GetStringAsync(url);
+			JArray array = JArray.Parse(json);
+			JToken latestRelease = array[0];
+
+			if (latestRelease["tag_name"] == null)
+				return;
+
+			Version release = Version.Parse(latestRelease["tag_name"].ToString());
+			Version current = Version.Parse(BuildInfo.Version);
+			
+			if (release > current)
+			{
+				MessageBox.Popup(
+					"Update Available", 
+					$"Update {release} is available.\nDo you want to open the releases page?",
+					MessageBoxViewModel.Buttons.YesNo,
+					UpdatePopupClosed);
+			}
+			else
+			{
+				MessageBox.Popup("No Updates Available",$"You are already on the latest version ({current})");
+			}
+		}
+
+		private void UpdatePopupClosed(MessageBoxViewModel.Result result)
+		{
+			if (result == MessageBoxViewModel.Result.Yes)
+			{
+				OpenUrl("https://github.com/ds5678/AssetRipper/releases");
+			}
+		}
+
+		// Called from UI
+		private void GithubClicked() => OpenUrl("https://github.com/ds5678/AssetRipper");
+
+		// Called from UI
+		private void WebsiteClicked() => OpenUrl(websiteURL);
+
+		private static void OpenUrl(string url)
+		{
+			if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
`Process.Start(url)` should be sufficient here - dotnet 6 is inherently cross-platform and knows what it's doing.

Reply:
I had created that method due to the issue at dotnet/runtime#17938 but changed it to Process.Start now.